### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by mann01**

### CODE REVIEW

File: content/participants/ttjc@mann01/index.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+type: "participant"
+name: "Mann Jain"
+portfolio: ""
+github-link: "https://github.com/mann01"
+linkedin-link: "https://linkedin.com/in/mann-jain-602bb9163"
+looking-for: "internship"
+description: "Web-Dev | ML | OpenCV | DS-Algo"
+---
+
+Hi! I am a Mann Jain ,3rd year undergraduate let me walk you through my intro, projects and blogs.
```
Comment (gkanishk) [inline content/participants/ttjc@mann01/index.md]: a Mann Jain?? Correct the typos

File: content/participants/ttjc@mann01/index.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+I know lyrics to almost all Bollywood songs.
+I am a good chess player
+
+### help I need?
+
+It would be good if someone tells me how to do auth for apps.
+
+### how to contact me?
+
+- LinkedIn ["https://linkedin.com/in/mann-jain-602bb9163"]
```
Comment (gkanishk) [inline content/participants/ttjc@mann01/index.md]: Use correct markdown format

File: content/participants/ttjc@mann01/index.md (hunk 1 above)
Comment (mann01) [reply content/participants/ttjc@mann01/index.md]: Ya I have modified the things in #276 please review!

**Comment by gkanishk** [comment]
Reviewing #276

**Pull request closed without merging**